monkey/monkey_island/cc/services/config.py

```diff
@@ -235,9 +239,34 @@ def init_default_config():
     def get_default_config(should_encrypt=False):
         ConfigService.init_default_config()
         config = copy.deepcopy(ConfigService.default_config)
+
         if should_encrypt:
             ConfigService.encrypt_config(config)
+
         logger.info("Default config was called")
+
+        return config
+
+    @staticmethod
+    def update_config_on_mode_set(mode: str) -> None:
```

Comment:
Let's move all of the methods added to a separate file. ConfigService is already 400 lines long...